### ISSUE TITLE
Feedback on page: /cloud/cdn/cloud-vcl-custom-snippets.html

### ISSUE THREAD

**Issue by ak77-prime** [magento/devdocs#8317]
# General issue

## Description:

Deleting of VCL snippets using the Magento Admin tool does not result in the snippet being removed at Fastly.

According to the Fastly extension [documentation](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/CUSTOM-VCL-SNIPPETS.md):
```
You can edit existing snippets in the UI. After any change you need to click Upload VCL to upload them to Fastly.

Please note: deleting snippets from the Magento UI only removes it from the Magento UI. If you really need a snippet gone you will need to delete it in the Fastly UI. Alternative to deleting a snippet is to set the VCL contents to empty content and upload VCL.**
```

Recommend adding a note/caution that deleting of snippets has to be done using the API.

## Possible solutions:

Explicitly provide instructions about deleting custom snippets in this section (**bold**):

```
You can create and manage custom VCL snippets from the Magento Admin UI or by using the Fastly API:

Magento Admin UI—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, **you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI**.
```

There's only a mention of deleting a specific VCL snippet further down in the article:

```
Delete an individual VCL snippet

* Get a list of snippets ...
```

**Comment by sathiyaaa** [comment]
@magento I am working on this

**Comment by bdenham** [comment]
Fixed with PR #8327
